src/ReleaseHistory.md

```diff
@@ -1,5 +1,7 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+* BREAKING: Move "transform" functionality into "rewrite" and delete redundant "transform" command
```

Comment:
>" [](start = 17, length = 1)

change from " to `.
Add the pull request number and link

Reply:
Done.

---
In reply to: [564879607](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r564879607) [](ancestors = 564879607)